components/PostCard.tsx

```diff
@@ -1,13 +1,43 @@
+import {
+  isPostADraft,
+  isPostInTheFuture,
+} from "helpers/checkOfDraftOrFuturePost";
 import { PostDocumentWithoutContent } from "interfaces";
 import Image from "next/image";
 import Link from "next/link";
 import { useAppDispatch } from "redux/typesHooks";
 
 import { setTags } from "../redux/slices/selectedTags";
+
 interface Props {
   post: PostDocumentWithoutContent;
 }
 
+export const DraftPostMark = () => (
+  <div className="draft-post">
+    <div className="triangle-draft triangle triangle-item">
+      <span className="triangle-text-draft triangle-text">draft</span>
+    </div>
+    <div className="small-triangles-draft">
+      <div className="small-triangles-item-draft-first triangle small-triangles-item-draft small-triangles-item"></div>
+      <div className="small-triangles-item-draft-second triangle small-triangles-item-draft small-triangles-item"></div>
```

Comment:
![image](https://user-images.githubusercontent.com/46556479/200105165-1f93d246-e7d0-4608-8613-321ce577e507.png)
Make the name of this class at least 2 or 3 times shorter,
90 characters is overkill